README.md

```diff
@@ -759,3 +759,61 @@ github:
 4. Configure Callback URL and Setup URL (with redirect on update set) to your host: `http${SECURE:+s}://${HOST_ADDRESS}/github`
 
 5. Configure Webhook URL to `http${SECURE:+s}://${HOST_ADDRESS}/_github/api/webhook` with the secret `secret`
+
+## Telegram Bot Service
+
+Telegram Bot Service is responsible for sending notifications from Huly to Telegram (private chats or groups). The service works as an external notification channel and does not provide interactive control or command execution for Huly.
+
+### Prerequisites
+
+1. Create a Telegram bot using **@BotFather**.
+2. Obtain the **BOT_TOKEN**.
+3. Determine the **CHAT_ID**:
+   - for private messages — send a message to the bot and retrieve `chat_id` using the `getUpdates` method
+   - for group chats — add the bot to the group and use the group `chat_id` (usually a negative value)
+
+### Docker Configuration
+
+1. Add the `telegram-bot` service to `compose.yml`:
+
+```yaml
+  telegram-bot:
+    image: hardcoreeng/telegram-bot:${HULY_VERSION}
+    container_name: telegram-bot
+    restart: unless-stopped
+    environment:
+      - PORT=4020
+      - DB_URL=${CR_DB_URL}
+      - BOT_TOKEN=${TELEGRAM_BOT_ID}
+      - ACCOUNTS_URL=http://account:3000
+      - SECRET=${SECRET}
+      - SERVICE_ID=telegram-bot-service
+      - STORAGE_CONFIG=minio|minio?accessKey=minioadmin&secretKey=minioadmin
+      - QUEUE_CONFIG=redpanda:9092
+      - QUEUE_REGION=
+    networks:
+      - huly_net
+```
+
+2. Configure the `front` service:
+
+```yaml
+front:
+  ...
+  environment:
+    - TELEGRAM_URL=http${SECURE:+s}://${HOST_ADDRESS}/_telegram
```

Comment:
I assume `TELEGRAM_BOT_URL` should be used here
`TELEGRAM_URL` is URL for Telegram integration service which synchronizes messages between Telegram and Huly